docs/requirements.txt

```diff
@@ -1 +1,2 @@
-sphinx==1.8.5
\ No newline at end of file
+sphinx==1.8.5
```

Comment:
@sharifsalah Should we consider upgrading the sphinx version? I notice we've fallen behind a bit. I'll defer to your guidance though.

Reply:
Thanks @gregdenton, yes I think so. I think I'd left it at 1.8.5 as that was the most recent version supported in Python 2.7.

I just tried setting the version of Sphinx to 3.0.3 which generates the docs as expected but it causes one of the current checks to fail. So if there aren't any objections, I'll submit this PR as-is and then I'll create a new issue to update the CI tests to support Sphinx 3.0.3.